extensions/PhabBugz/lib/User.pm

```diff
@@ -199,5 +200,27 @@ sub get_phab_bugzilla_ids {
   return \@results;
 }
 
+#################
+#   Helpers     #
+#################
+
+sub enable_user {
+  my ($self) = @_;
+  my $params = {
+    objectIdentifier => $self->phid,
+    transactions     => [{type => 'disabled', value => false,}]
+  };
+  request('user.edit', $params);
+}
+
+sub disable_user {
+  my ($self) = @_;
+  my $params = {
+    objectIdentifier => $self->phid,
+    transactions     => [{type => 'disabled', value => true,}]
+  };
+  request('user.edit', $params);
+}
```

Comment:
Nit: I might join these into a `sub set_user_enabled_status($enabled)`, then smooth out the call site as well to collapse the `if/else`.

Reply:
Will do that.